lib/Crypto/keypairgenerationparameters.cpp

```diff
@@ -219,6 +219,8 @@ EcKeyPairGenerationParameters::EcKeyPairGenerationParameters()
     : KeyPairGenerationParameters()
 {
     setKeyPairType(KeyPairGenerationParameters::KeyPairEc);
+    // TODO: is this a good default choice?
+    setEllipticCurve(CryptoManager::CurveSecp521r1);
```

Comment:
I'm not sure how many web-browsers etc support the NIST-P521 curve (aka SECP521r1), perhaps NIST-P384 or NIST-P256 would be better default choices.